### ISSUE TITLE
Request to add "clear steam achievement" support

### ISSUE THREAD

**Issue by NoSoul8** [greenheartgames/greenworks#240]
Would you kindly add "Deactivate achievement" action? It is often used for testing and debugging steam achievements, especially with tricky conditions to trigger. So if something goes wrong (e.g. achievement gets activated, although it should not) we have an option to quickly drop achievement, make amends to events and try again. Super useful near release, while polishing Steam stuff before sending a build to moderation.

My request may be related to https://github.com/greenheartgames/greenworks/issues/23

**Comment by MikalDev** [comment]
FYI it may not be exactly what you want, but I use the ClearStats function for that in testing (the function clears stats AND achievements, but not individual achievements.) it would be nice to have something that clears individual achievements.

**Comment by NoSoul8** [comment]
Yes, clearing individual achievement is what i need most of steam features.
For testing and retesting.

**Comment by MikalDev** [comment]
I was about to start work on this, but it looks like it's already there? Have you tried the below?

greenworks.clearAchievement(achievement, success_callback, [error_callback])
achievement String - The achievement needs to be cleared
success_callback Function()
error_callback Function(err)

**Comment by MikalDev** [comment]
FYI - I forgot I had used this in the past in my Construct 3 Greengrinds plugin. It works for me in my test app with Electron.
